polymorphic_auth/usertypes/email/apps.py

```diff
@@ -6,8 +6,12 @@
 # See: https://docs.djangoproject.com/en/1.8/ref/applications/#django.apps.AppConfig.ready
 
 from django.apps import AppConfig
+from django.utils.module_loading import autodiscover_modules
 
 
 class AppConfig(AppConfig):
     name = 'polymorphic_auth.usertypes.email'
     label = 'polymorphic_auth_email'
+
+    def ready(self):
+        autodiscover_modules('plugin')
```

Comment:
I think the `autodiscover_modules` call should go in `polymorphic_auth.apps.AppConfig.ready()` instead of here, and the argument should be `plugins` (plural) or maybe even `polymorphic_auth_plugins` to avoid module name clash with any other apps.